doc/sphinx/how-to/how-to-integrate-a-custom-wayland-protocol.md

```diff
@@ -0,0 +1,137 @@
+---
+discourse: 11993
+---
+
+# How to integrate a custom Wayland protocol
+
+With the [Mir 1.3 release](https://discourse.ubuntu.com/t/mir-1-3-0-release/11638/2)
```

Comment:
Mir 1.3 is over 6 years old, I think all consumers will be using this or greater by now.

Reply:
Yes, the oldest version of Mir that needs _any_ thought is 1.8. That is still used downsteam by Ubuntu Touch but, as they maintain their own fork, we seldom care